### PR TITLE
IBX-704: Added LocationChildrenTermAggregation

### PR DESCRIPTION
**Pull request by mateuszbieniek**
| Question                                  | Answer
| ---------------------------------------- | ------------------
| **JIRA issue**                          | https://issues.ibexa.co/browse/IBX-704
| **Type**                                   | improvement
| **Target eZ Platform version** | `v3.2`
| **BC breaks**                          | no
| **Doc needed**                       | yes

This is the kernel part for https://github.com/ezsystems/ezplatform-admin-ui/pull/1792

### CODE REVIEW

**Comment by mateuszbieniek** [comment]
ping @ezsystems/engineering-team

**Comment by nkostiv** [comment]
> ping @ezsystems/engineering-team

@mateuszbieniek engineering-team includes also java team. We have separate `@ezsystems/php-dev-team`, `@ezsystems/javascript-dev-team`,` @ezsystems/java-dev-team `

**Comment by mateuszbieniek** [comment]
> > ping @ezsystems/engineering-team
> 
> @mateuszbieniek engineering-team includes also java team. We have separate `@ezsystems/php-dev-team`, `@ezsystems/javascript-dev-team`,`@ezsystems/java-dev-team`

Awww, my bad - sorry.

File: eZ/Publish/API/Repository/Values/Content/Query/Aggregation/LocationChildrenTermAggregation.php [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\API\Repository\Values\Content\Query\Aggregation;
+
+final class LocationChildrenTermAggregation extends AbstractTermAggregation
```
Comment (adamwojs) [inline eZ/Publish/API/Repository/Values/Content/Query/Aggregation/LocationChildrenTermAggregation.php:11]: Integration tests are missing.

**Review by adamwojs — CHANGES_REQUESTED** [review]

**Review by adamwojs — CHANGES_REQUESTED** [review]

**Review by alongosz — APPROVED** [review]

**Review by adamwojs — APPROVED** [review]

**Review by webhdx — APPROVED** [review]

**Review by bogusez — APPROVED** [review]

**Comment by mateuszbieniek** [comment]
Merged up
1.3: https://github.com/ezsystems/ezplatform-kernel/commit/d8d00233c2815ea24e1489b25abc220aac70b342
master: https://github.com/ezsystems/ezplatform-kernel/commit/1945b15ecbb0f008174aaa091ace4faf979e5250

**Pull request merged**